pkg/azureutils/azure_disk_utils.go

```diff
@@ -698,13 +698,40 @@ func IsValidAccessModes(volCaps []*csi.VolumeCapability) bool {
 	return foundAll
 }
 
+func IsMultiNodeAzVolumeCapabilityAccessMode(accessMode v1alpha1.VolumeCapabilityAccessMode) bool {
+	return accessMode == v1alpha1.VolumeCapabilityAccessModeMultiNodeMultiWriter ||
+		accessMode == v1alpha1.VolumeCapabilityAccessModeMultiNodeSingleWriter ||
+		accessMode == v1alpha1.VolumeCapabilityAccessModeMultiNodeReaderOnly
+}
+
+func HasMultiNodeAzVolumeCapabilityAccessMode(volCaps []v1alpha1.VolumeCapability) bool {
+	for _, volCap := range volCaps {
+		if IsMultiNodeAzVolumeCapabilityAccessMode(volCap.AccessMode) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func IsMultiNodePersistentVolume(pv v1.PersistentVolume) bool {
+	for _, accessMode := range pv.Spec.AccessModes {
+		if accessMode == v1.ReadWriteMany || accessMode == v1.ReadOnlyMany {
```

Comment:
@abhisheksinghbaghel I included `v1.ReadOnlyMany` in the modes from which we exclude replicas, but I noticed the design doc only calls out `v1.ReadWriteMany`. Was it intentional that only RWX volumes would be excluded? Is there a use case for replicas with ROM volumes?

Reply:
I sync'ed offline with @abhisheksinghbaghel. His response was:

> For your GitHub comment, I think it's ok to ignore replicas for read only many. It is probably a miss from my side at the time of writing the doc.